package.json

```diff
@@ -110,5 +114,14 @@
     "url": "https://github.com/paypal/downshift/issues"
   },
   "homepage": "https://github.com/paypal/downshift#readme",
-  "dependencies": {}
+  "flow-coverage-report": {
+    "includeGlob": [
+      "test/**/*.js",
+      "test/**/*.jsx"
```

Comment:
I prefer to not use `.jsx` as an extension and just use `.js` for everything. Could you make that change? I realize that the TS files have the `.tsx` extension, but that wasn't my choice and I try to stay out of TS decisions. Thanks :)

Reply:
This is what I get when I switch from JSX to JS

```
downshift/test/custom.test.js
 6:8  error  Unable to resolve path to module 'downshift'  import/no-unresolved
39:5  error  Unexpected console statement                  no-console
```